guide/getting-started/1-getting-started.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: guide
+title: Getting started
+nav_order: 1
+has_children: true
+permalink: /getting-started/introduction/
+---
+
+# Design Bitcoin for everyone
+
+Wouldn’t it be great if we could send money to anyone in the world as easily as we can send emails and text messages? That’s one of the promises of Bitcoin. Since it was [first introduced in 2009](https://bitcoin.org/bitcoin.pdf){:target="_blank"}, it has become a globally recognized money and a public good. As it has grown in importance and its use has become more diverse, the need to make it more usable and more useful for more people across the world has increased. Since bitcoin was designed in a way that there is no central entity that defines what it should be like, each of us can help shape it. Many already have, as is reflected in the thousands of projects and businesses that already form a vibrant ecosystem. With this guide, we hope to provide information, background, best practices and resources for anyone who wants to design a product or service that uses bitcoin.
+
+## By the people, for the people
+
+One of the unique aspects of Bitcoin is that it is an open-source project and no single group, company or government owns or controls it. That means all of us can freely participate and contribute as much or as little as we want. All contributions are equally valid, and there are no gatekeepers. This decentralization is an incredibly important property for a global currency as it provides a level playing field. It also means that there are many different people independently working on different projects, and that we need to help each other to make Bitcoin as useful for as many people as possible.
```

Comment:
I'd be hesitant to call it a `global currency` as I feel it implies it will be used as currency is today and there is no guarantee of that given the prevailing digital gold narrative. I'd be okay with `global transfer of value` maybe?

Reply:
Different people have different aspirations for bitcoin. My own opinion and one I advocate for with this community is to aspire to become a global currency. I think we should design with that in mind. This is not inconsistent with bitcoin today as speculative investment to become a SoV or digital gold - in fact IMO it is a necessary step to become a global currency. I literally do not see how it could be a global currency if it did not go through a monetization phase of $0 to something on the order of trillions of dollars of market cap (based on the size of M0 or the size of gold).

As long as we are not overpromising that _today's_ bitcoin is a usable global currency then I have no issue with us using that language and working toward it.

Reply:
:) thanks for the input. I've rewritten this page already multiple times and expect to do so again. It's a tricky one to get right. One option for this particular phrasing of global currency is to describe bitcoin as something new with a unique combination of properties that allows it be used in different ways, and that these uses are expanding as it gets adopted more widely. Which I think is also why there is a much bigger cryptocurrency/altcoin ecosystem, people are trying to figure out what else is possible with this new thing.

Reply:
I ended up removing this copy when restructuring the guide hierarchy.